src/corelibs/U2Designer/src/DelegateEditors.h

```diff
@@ -352,6 +357,7 @@ private slots:
 
 protected:
     QVariantMap items;
+    QMap<QString, QString> visibleKeyToBusValueMap;
```

Comment:
This name is misleading. The correct name is "busValueToDisplayNameMap": by the value in the bus (format-id here) you can get the display value for the combo-box: format name.